---
title: "Training Information"
layout: default
---

{% include navbar.html %}

# About

A hands-on training workshop on Post-GWAS Analysis Using SNP-Seek and RicePilaf. This in-depth session is designed for researchers looking to advance their skills in candidate gene analyses.

## Short Description:

* <a href="https://snpseek.irri.org/" target="_blank">SNP-Seek</a> version 3 is an online web application that provides a user-friendly, comprehensive database for the genetic variations, phenotypes, and variety information of the 3500 rice genomes accessions. <br>
    * Locedie Mansueto, Roven Rommel Fuentes, Frances Nikki Borja, Jeffery Detras, Juan Miguel Abriol-Santos, Dmytro Chebotarov, Millicent Sanciangco, Kevin Palis, Dario Copetti, Alexandre Poliakov, Inna Dubchak, Victor Solovyev, Rod A. Wing, Ruaraidh Sackville Hamilton, Ramil Mauleon, Kenneth L. McNally, Nickolai Alexandrov, <strong>Rice SNP-seek database update: new SNPs, indels, and queries</strong>, Nucleic Acids Research, Volume 45, Issue D1, January 2017, Pages D1075â€“D1081, <a href="https://doi.org/10.1093/nar/gkw1135" target="_blank">https://doi.org/10.1093/nar/gkw1135</a>

* <a href="https://ricepilaf.irri.org/" target="_blank">RicePilaf</a> is a web-based post-GWAS/QTL application that integrates pangenomic, coexpression, regulatory, epigenomic, ontology, pathway, and text-mining information. It aims to provide functional insights into rice QTLs and GWAS regions identified from QTL and GWAS mapping studies. Alternate servers: <a href="https://ricepilaf.ird.fr/" target="_blank">RicePilaf IRD</a> and <a href="https://ricepilaf.bioinfodlsu.com/" target="_blank">RicePilaf DLSU</a> <br>
    * Anish M S Shrestha, Mark Edward M Gonzales, Phoebe Clare L Ong, Pierre Larmande, Hyun-Sook Lee, Ji-Ung Jeung, Ajay Kohli, Dmytro Chebotarov, Ramil P Mauleon, Jae-Sung Lee, Kenneth L McNally, <strong>RicePilaf: a post-GWAS/QTL dashboard to integrate pangenomic, coexpression, regulatory, epigenomic, ontology, pathway, and text-mining information to provide functional insights into rice QTLs and GWAS loci</strong>, GigaScience, Volume 13, 2024, giae013, <a href="https://doi.org/10.1093/gigascience/giae013" target="_blank">https://doi.org/10.1093/gigascience/giae013</a>

## What You'll Learn:

* Navigate and utilize the SNP-Seek version 3 database

* Perform post-GWAS analysis workflows using RicePilaf

* Best practices for downstream functional analysis

## What to Bring: 

* Please bring your laptop. All datasets will be provided.

## For more information on the Bioinformatics team:

* <a href="https://bioinfo.irri.org/" target="_blank">Bioinformatics Unit</a>, Breeding Innovations and Informatics Cluster, Rice Breeding Innovations Department, International Rice Research Institute
